homeassistant/components/schlage/quality_scale.yaml

```diff
@@ -0,0 +1,84 @@
+rules:
+  # Bronze
+  config-flow: done
```

Comment:
Also, `already_configured` is referencing `[%key:common::config_flow::abort::already_configured_device%]` which is about `device` we also have `[%key:common::config_flow::abort::already_configured_account%]`